docs/design_flaws.md

```diff
@@ -355,3 +442,70 @@ GetNumberedTMHM: ; d417
 	ld c, a
 	ret
 ```
+
+**Fix:**
+
+Move `ITEM_C3` and `ITEM_DC` above all the TMs in every table of item data.
```

Comment:
These are catch rates used in RBY. You'll need to also modify the time capsule code to compensate.